test/alternator/test_describe_table.py

```diff
@@ -112,12 +120,73 @@ def test_describe_table_item_count(test_table):
     got = test_table.meta.client.describe_table(TableName=test_table.name)['Table']
     assert 'ItemCount' in got
 
-# Similar test for estimated size in bytes - TableSizeBytes - which again,
-# may reflect the size as long as six hours ago.
-def test_describe_table_size(test_table):
-    got = test_table.meta.client.describe_table(TableName=test_table.name)['Table']
-    assert 'TableSizeBytes' in got
-    assert got['TableSizeBytes'] >= 0
+def test_describe_table_size_with_timeout_set_to_0(cql, dynamodb):
```

Comment:
Please don't delete the old test - it's still a valid test, and runs on both Scylla and DynamoDB - unlike your new test.

Reply:
You add the scylla_only faux-fixture.
When you do that, please add a comment explaining why this is a Scylla-only test.